src/components/atoms/PamphletParagraph/PamphletParagraph.jsx

```diff
@@ -1,53 +1,98 @@
 import PropTypes from 'prop-types';
+import Image from 'next/image';
 import { Box, Divider, Typography } from '@mui/material';
 import { ButtonLink } from 'components/atoms';
 import { applyAlpha, designColor } from 'theme/palette';
+import { typesNextImage } from 'model/view/component/prop-types';
 
 const DividerGrey = () => (
-  <Divider sx={{ borderColor: applyAlpha('40', designColor.neroGray) }} />
+  <Divider
+    sx={{ borderColor: applyAlpha('66', designColor.neroGray), mx: { lg: 15 } }}
+  />
 );
 
 export default function PamphletParagraph({
   title,
   variant,
+  img,
   body,
   button,
-  hasDividerBottom,
+  hasDivider = false,
+  sx = {},
 }) {
   return (
-    <>
-      <Typography sx={{ my: 7 }} variant={variant} textAlign="center">
-        {title}
+    <Box sx={sx}>
+      {hasDivider && <DividerGrey />}
+
+      <Typography textAlign="center" sx={sxTitleMargin} variant={variant}>
+        {PrettyWrapSentence(title)}
       </Typography>
 
+      {img && (
+        <Box textAlign="center" sx={sxParagraphMargin}>
+          <Image alt="" {...img} style={{ borderRadius: '5px' }} />
+        </Box>
+      )}
+
       {body &&
         body.map((val, index) => (
-          <Typography sx={{ mb: 7 }} variant="body1" key={'ppBody' + index}>
+          <Typography
+            variant="body1"
+            sx={sxParagraphMargin}
+            key={index + '_PamphletParagraph'}
```

Comment:
Non blocking question: here and on line 89 - is there any reason not to use a template string? I think that they're much cleaner than concatenating string.

Reply:
done